lib/kits/events/tracker.ts

```diff
@@ -239,14 +242,14 @@ class Tracker {
         };
 
         if (this.eventsTrackUrl) {
-            dispatch(this.eventsTrackUrl, this.eventsTrackHeaders);
+            this.dispatch();
             this.hasMadeInitialDispatch = true;
```

Comment:
This on-instatiation dispatching and tracking of `hasMadeInitialDispatch` we can also get rid of if there aren't events(from a persisted pool) to dispatch immediately when `new Tracker`, but we would want to cover that for a transition for any backed-up event pool for a week or whatever